src/client/Microsoft.Identity.Client/ManagedIdentity/EnvironmentVariables.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http.Headers;
+
+namespace Microsoft.Identity.Client.ManagedIdentity
+{
+    internal class EnvironmentVariables
+    {
+        public static string Username => Environment.GetEnvironmentVariable("AZURE_USERNAME");
```

Comment:
In which azure resource username and password is used for MI?

Reply:
I think this can be removed. This class is also copy pasted. Will cleanup after we have moved all the apps